core/idl/wallet/configuration.djinni

```diff
@@ -66,6 +66,8 @@ Configuration = interface +c {
     const BLOCKCHAIN_EXPLORER_ENGINE: string = "BLOCKCHAIN_EXPLORER_ENGINE";
     # Sets the API endpoint for API based engine (Ledger's API).
     const BLOCKCHAIN_EXPLORER_API_ENDPOINT: string = "BLOCKCHAIN_EXPLORER_API_ENDPOINT";
+    # Sets the node's endpoint to broadcast transactions with one's node
+    const BLOCKCHAIN_NODE_ENDPOINT: string = "BLOCKCHAIN_NODE_ENDPOINT";
```

Comment:
Seems fine to me, but is this used only for Bitcoin? Does it make sense to use the overall configuration for that?